archive.php

```diff
@@ -0,0 +1,25 @@
+<?php
```

Comment:
@kienstra Thanks for thinking of this! Added a specific check for `adventure` post type to avoid showing the non-existing rating in case of `post`s if they accept comments.